src/sql/base/browser/ui/adsWidget.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Widget } from 'vs/base/browser/ui/widget';
+``
+/**
+ * This interface is implemented by SelectBox and InputBox to provide a common API surface area in connectionWidget.ts
+ * If more widgets must be used in Connection Dialog, they should implement this interface.
+ */
+export interface AdsWidget extends Widget {
+
+	get value(): string;
+
+	getId(): string;
```

Comment:
lets be consistent by using getter for id and aria-label. if those methods were already implemented, I think we should correct them.

Reply:
I changed that for id, but ariaLabel property is private in InputBox and getAriaLabel() is implemented in vs for inputBox.